### ISSUE TITLE
allow blank email

### ISSUE THREAD

**Issue by badiku** [go-gitea/gitea#21704]
### Feature Description

hi, for decades we never use email.  why every account should have an email ?

I wish email is not a required field when create a new user.

I modified custom\templates\admin\user\new.tmpl but it seems that the requirement is deep in server backend.

please add an option to disable email requirement, or can someone kindly point to me where to modify it by myself.

**Comment by lunny** [comment]
Since git requires an email, I don't think an email address could be ignored?

**Comment by badiku** [comment]
I trid to remove some valid function, so it can transfer register data with blank email  from web to backend, but still failed:

> [ERROR] [6368b1b7] CreateUser: e-mail already in use [email: ]

so it is deeply required by some code. 

> Since git requires an email, 

but git allow blank email. for years I always begin my new git environment by this :  git config --global  user.email ""

and there is no problem at all.

if you use git as a standard, I think we should allow blank email, just like git does.


the fact is , nowadays email is never used, except for register github.

specially , for internal team usage , there is no need to fore use email.

**Comment by badiku** [comment]
I trid to remove some valid function, so it can transfer register data with blank email from web to backend, but still failed:

`    [ERROR] [6368b1b7] CreateUser: e-mail already in use [email: ]`

so it is deeply required by some code.

>     Since git requires an email,

but git allow blank email. for years I always begin my new git environment by this : git config --global user.email ""

and there is no problem at all.

if you use git as a standard, I think we should allow blank email, just like git does.

the fact is , nowadays email is never used, except for register github.

specially , for internal team usage , there is no need to fore use email.

**Comment by lunny** [comment]
You can use false email address like myname@myorg.com and disable email confirmation in app.ini

**Comment by badiku** [comment]
yeah, I already did this. just tired to type a fake email for each person. 

 I'd like to change it by myself. but seems a little complicated...

**Comment by lunny** [comment]
This is too complicated and I couldn't find more benefit to implementing it.

**Comment by Mosch0512** [comment]
how to disable email conformation in app ini.
Personaly i would need that feature too.
But only from the admins perspective

**Comment by MaxEden** [comment]
Using gitea for internal repo manager is kinda hard right now. There are no private invites so I have to create user accounts manually, so I have to make up email addresses. It's not elegant to put it mildly
